codecarbon/output_methods/http.py

```diff
@@ -41,6 +41,7 @@ class CodeCarbonAPIOutput(BaseOutput):
     run_id = None
 
     def __init__(self, endpoint_url: str, experiment_id: str, api_key: str, conf):
+        self.use_emissions_delta = True
```

Comment:
On est sûrs de vouloir ça par défaut ? ça va pas changer le comportement chez nos users ?

Reply:
You are right, sorry this was coming from an implementation before the composite output handling, and I forgot to remove it. Now it is done ==> [063b7f8](https://github.com/mlco2/codecarbon/pull/542/commits/063b7f86af919aa39e91c40267e052f552c800d5)